tests/test_general.py

```diff
@@ -68,6 +69,68 @@ def test_tilt_desired(self):
         ].empty
 
 
+class TestRTLHeight:
+    # The return to home height changes with the distance from home
+    # mode was triggered
+    # check the height above ground while the drone returns to home. compare it with 
+    # the allowed maximum or minimum heights, until the drone has reached home and motors have been turned off
+
+   def setup_class(self):
+        # get the required data
+        filepath = getfilepath()
+        topics = [
+            "vehicle_local_position",
+            "vehicle_status",
+        ]
+        self.ulog = pyulog.ULog(filepath, topics)
+        self.df = ulogconv.merge(ulogconv.createPandaDict(self.ulog))
+
+
+   def test_rtl(self):
+       # TODO adjust this test to also work with cone (only for yuneec)
+        NAVIGATION_STATE_AUTO_RTL = 5           # see https://github.com/PX4/Firmware/blob/master/msg/vehicle_status.msg
+        max_height_within_RTL_MIN_DIST = 10     # within this radius, the drone should not go higher than this value after RTL has been triggered
```

Comment:
Is this `10` hard coded in the firmware? Or how did you come up with that number?